qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -108,7 +108,8 @@ def get(self):
         # parse it
         realdata = np.fromstring(rawdata, dtype='<i2')
         numbers = realdata[::2]*2.0**(realdata[1::2]-124)
-
+        if self.shape[0] != N:
```

Comment:
This looks like a good precaution (and I guess it's indeed related to fast measurements with the QDac), but shouldn't it be a seperate PR?

Reply:
Maybe it should but the commits have already been squashed because its a backport and I don't really see the point of splitting them

Reply:
Fair enough.